Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -126,22 +126,21 @@ private extension CardPresentPaymentStore {
             // Note: Here we transition from the Stripe Terminal Payment Intent
             // to the WCPay backend managed Payment Intent, which we need
             // to use to capture the payment.
-
-            self.remote.captureOrderPayment(for: siteID, orderID: orderID, paymentIntentID: intent.id) { result in
-                switch result {
-                case .success(let intent):
-                    guard intent.status == .succeeded else {
-                        DDLogDebug("Unexpected payment intent status \(intent.status) after attempting capture")
-                        onCompletion(.failure(CardReaderServiceError.paymentCapture()))
-                        return
-                    }
-                    // This is always failing at this point. 
-                    //onCompletion(.success(receiptParameters))
-                case .failure(let error):
-                    onCompletion(.failure(error))
-                    return
-                }
-            }
+            // This is always failing at this point. Commented out for now.
```

Comment:
Commenting out until [this](https://github.com/Automattic/woocommerce-payments/tree/try/capture-endpoint) is in